### PR TITLE
Add rust-src

### LINKED ISSUE #160: Include rust-src
Without rust-src it's impossible to use rust-analyzer

### PR DESCRIPTION
**Pull request by vilgotf**
closes #160

### CODE REVIEW

**Comment by vilgotf** [comment]
Works for me!

File: org.freedesktop.Sdk.Extension.rust-nightly.yml [hunk 1]
```diff
@@ -31,15 +32,25 @@ modules:
...
         x-checker-data:
           type: rust
           package: rust
           channel: nightly
           target: x86_64-unknown-linux-gnu
+      - type: archive
+        dest: rust-src
+        url: https://static.rust-lang.org/dist/2021-05-14/rust-src-nightly.tar.xz
+        sha256: 0a17d813d5df512b222941e2363b5a04a5f5e34c7bacf5bdda4d30497a350fc0
+        x-checker-data:
```
Comment (bilelmoussaoui) [inline org.freedesktop.Sdk.Extension.rust-nightly.yml:47]: You need to specify that you want nightly rust-src

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]

File: org.freedesktop.Sdk.Extension.rust-nightly.yml (hunk 1 above)
Comment (vilgotf) [reply org.freedesktop.Sdk.Extension.rust-nightly.yml:45]: Where do you want this to be specified? As part of  the "dest" name? (rust-src-nightly)?

Comment (bilelmoussaoui) [reply org.freedesktop.Sdk.Extension.rust-nightly.yml:45]: As part of the x-checker, see other sources for an example

**Comment by bilelmoussaoui** [comment]
Thanks for your contribution! :)

**Pull request merged**